src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -18,22 +18,10 @@ function focusForm() {
   focusElement('.nav-header > h2');
 }
 
-const scroller = Scroll.scroller;
-const scrollToTop = () => {
-  scroller.scrollTo(
-    'topScrollElement',
-    window.Forms?.scroll || {
-      duration: 500,
-      delay: 0,
-      smooth: true,
-    },
-  );
-};
```

Comment:
Can you describe the differences between these two `scrollToTop` functions?

https://github.com/department-of-veterans-affairs/vets-website/blob/faf2613a8ac774708f7850b14d7b019ad3895985/src/platform/utilities/ui/scrollToTop.js#L3-L12

Also, this old version is passing in an object if `window.Forms?.scroll` doesn't exist, and its replacement is passing in an empty string instead. What is the impact of that change?

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/18770#discussion_r711345683 Going to link to this comment on the other PR as the explanation is the same I believe.

Reply:
I missed the first part of your question above the code block - these were both intended to be converted and I totally overlooked the first one. I made that correction in my PR now to keep them both the same.